modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -15,23 +15,8 @@ class {{{classname}}} {
     {{#maximum}}
   // maximum: {{{maximum}}}
     {{/maximum}}
-  {{{dataType}}} {{{name}}};
-  {{/isEnum}}
-  {{#isEnum}}
-    {{#isContainer}}
-      {{#isArray}}
-  List<{{{classname}}}{{{enumName}}}> {{{name}}};
-      {{/isArray}}
-      {{#isMap}}
-  Map<String, dynamic> {{{name}}};
-      {{/isMap}}
-    {{/isContainer}}
-    {{^isContainer}}
-      {{#allowableValues}}
-  {{{classname}}}{{{enumName}}} {{{name}}}{{#required}}{{#defaultValue}} = {{{classname}}}{{{enumName}}}._({{{defaultValue}}}){{/defaultValue}}{{/required}};
-      {{/allowableValues}}
-    {{/isContainer}}
   {{/isEnum}}
+  {{{datatypeWithEnum}}} {{{name}}};
```

Comment:
I am not sure about all of this change, especially the enum default value. Currently all default values are set in the constructor. As far as I can tell in that case a wrong constructor default value would be generated.

Reply:
I think the default value should stay as null, unless field is marked as non-nullable, required or has default value provided, this looks good to me.